README.md

```
@@ -23,3 +23,38 @@ make dev
+
+#### WhatsApp Automation
+The WhatsApp automation is still a very manual process. There are two scripts:
+- `contrib/whats-extract.js`: extracts the messages from WhatsApp, format them,
+  and output to the stdout
+- `contrib/import.py`: parse the messages using LLM, ask for confirmation, and
+  update the database.
+
+Running:
+
+1.
+    ```shell
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)